### PR TITLE
More preparations for PyPI release

### PR DESCRIPTION
**Pull request by bmerry**
- Use katversion
- Update email address (@sratcliffe: alias required)
- Add Trove classifiers
- Update installation instructions
- Add a changelog

### CODE REVIEW

**Comment by bmerry** [comment]
Don't review yet. This is work in progress, and I'm just creating a draft PR as a reminder to myself.

**Comment by bmerry** [comment]
@sratcliffe I've asked Ludwig to review but can you add the email alias?

File: setup.py [hunk 1]
```diff
@@ -46,6 +49,7 @@ def __str__(self):
 setup(
     name="katsdpimager",
     description="GPU-accelerated radio-astronomy spectral line imager",
+    long_description=long_description,
```
Comment (mmphego) [inline setup.py]: In order to add `README.md` content on PyPi properlly, Perhaps also add the [description-content-type](https://packaging.python.org/specifications/core-metadata/#description-content-type).

See: https://packaging.python.org/guides/making-a-pypi-friendly-readme/#including-your-readme-in-your-package-s-metadata

Comment (bmerry) [reply setup.py]: The default content type for long_description is reST, so it's not necessary to specify unless you're using a different content type like markdown.

Comment (mmphego) [reply setup.py]: Learn something new every day. :wink:

File: Jenkinsfile [hunk 2]
```diff
@@ -16,4 +16,4 @@ katsdp.standardBuild(
     python2: false,
     prepare_timeout: [time: 90, unit: 'MINUTES'],
     test_timeout: [time: 90, unit: 'MINUTES'])
-katsdp.mail('bmerry@ska.ac.za')
+katsdp.mail('sdpdev+katsdpimager@ska.ac.za')
```
Comment (ludwigschwardt) [inline Jenkinsfile]: That's a nifty idea!

**Pull request merged**